lib/TokenManager.js

```diff
@@ -14,7 +14,7 @@
 /* eslint complexity:[0,8] max-statements:[0,21] */
 var util = require('./util');
 var AuthSdkError = require('./errors/AuthSdkError');
-var storageUtil = require('./storageUtil');
+var storageUtil = require('./browser/browserStorageUtil');
```

Comment:
As improvement for next step, feels like we shall move all browser specific feature into `browser` subfolder. ?

Reply:
The OIDC functionalities _might_ be even moved in another library in future, so for now we leave them there until we have a more clear plan 👍 
cc: @jmelberg-okta